src/inference/include/openvino/runtime/intel_cpu/properties.hpp

```diff
@@ -63,5 +63,11 @@ static constexpr Property<bool> denormals_optimization{"CPU_DENORMALS_OPTIMIZATI
  */
 static constexpr Property<float> sparse_weights_decompression_rate{"CPU_SPARSE_WEIGHTS_DECOMPRESSION_RATE"};
 
+/**
+ * @brief Defines how many records can be stored in the CPU runtime parameters cache per CPU runtime parameter type per
+ * stream.
+ */
+static constexpr Property<int32_t, PropertyMutability::RW> cpu_runtime_cache_capacity{"CPU_RUNTIME_CACHE_CAPACITY"};
```

Comment:
why does it become public API?
I suppose these properties should be part of CPU plugin implementation, because they are not exposed outside

Reply:
> intel_cpu/properties.hpp, isn't it?

it's a public API, while old properties (config) were a part of dev API.

> @ilya-lavrenov am i right?

Yes, your categorization looks right.

Reply:
Update properties as above policy.